fixtures/mock-oauth2-provider/package.json

```diff
@@ -33,12 +33,7 @@
   "dependencies": {
     "passport": "^0.4.1",
     "tslib": "^1.11.2",
```

Comment:
The latest version should be `2.0.0` now.

Reply:
@raymondfeng all loopback packages refer only tslib 1.12.0 as of now

Reply:
No, you need to rebase to master. It's all `2.0.0` now.